server/service.go

```diff
@@ -419,7 +421,7 @@ func (m *BoostService) handleGetHeader(w http.ResponseWriter, req *http.Request)
 			}
 
 			// Use this relay's response as mev-boost response because it's most profitable
-			log.Debug("received a good bid")
+			log.Debug("selecting bid")
```

Comment:
at this point it's not selected. it's just better than a previous one. i'd leave that unchanged, or reword somehow differently.

Reply:
Oh, I was wording that based on the comment above that line. Does that mean that the comment is wrong, or that "selecting" means something other than "this is what we will respond with"?

The wording "than a previous one" implies there was a previous one, which is not necessarily the case right?